sammba/registration/struct.py

```diff
@@ -426,75 +456,32 @@ def anats_to_common(anat_filenames, write_dir, brain_volume,
     out_tstat_warp_head = tstat(in_file=out_tcat.outputs.out_file,
                                 outputtype='NIFTI_GZ')
 
-    ###########################################################################
```

Comment:
I am not following: why is this removed ?

Reply:
there was no purpose to having code just for the second iteration

Reply:
but the warp_file computation is different no ?